tests/robot-tests/tests/libs/dates_and_times.py

```diff
@@ -0,0 +1,49 @@
+import datetime
+import os
+import pytz
+from tests.libs.selenium_elements import sl
+
+
+def get_london_date(offset_days: int = 0, format_string: str = "%-d %B %Y") -> str:
+    return _get_date_and_time(offset_days, format_string, "Europe/London")
+
+
+def get_london_date_and_time(offset_days: int = 0, format_string: str = "%-d %B %Y") -> str:
+    return _get_date_and_time(offset_days, format_string, "Europe/London")
+
+
+def get_local_browser_date_and_time(offset_days: int = 0, format_string: str = "%-d %B %Y") -> str:
+    return _get_date_and_time(offset_days, format_string, _get_browser_timezone())
+
+
+def get_london_day_of_month(offset_days: int = 0) -> str:
+    return get_london_date_and_time(offset_days, "%-d")
+
+
+def get_london_month_date(offset_days: int = 0) -> str:
+    return get_london_date_and_time(offset_days, "%-m")
+
+
+def get_london_month_word(offset_days: int = 0) -> str:
+    return get_london_date_and_time(offset_days, "%B")
+
+
+def get_london_year(offset_days: int = 0) -> str:
+    return get_london_date_and_time(offset_days, "%Y")
```

Comment:
Not suggesting a change, but would be interested in discussing the pros and cons of this kind of approach with you at some point. 

I think my general inclination is the reverse of yours, which isn't to wrap things up like this, but in this case I can see it makes sense. But I've seen other instances where more complex things than this gets wrapped up with several layers of abstraction. If it breaks or needs updating, understanding what it is doing can be quite difficult, especially if it's already been tweaked a couple of times previously. In those cases I've tended to inline all the abstraction so I could understand it better, and I've frequently found that has allowed me to simplify things a lot.